### DISCUSSION TITLE
How can I have stylix and switch between dark and light theme ?

### DISCUSSION THREAD

**Discussion by ilyasturki** [nix-community/stylix#525] [Q&A]
For example on gnome you have a dark mode toggle button, but it doesn't work with stylix because the gtk theme for example will be fixed for either dark or light mode. There's also a setting for polarity, but it's also fixed for dark or light. Is there a workaround into having dark light theme toggle ?

**Comment by trueNAHO** [comment]
There are application limitations making a universal Stylix solution impossible.

Related: https://github.com/danth/stylix/discussions/371

I opened issue https://github.com/danth/stylix/issues/530 to address this question in the documentation.
